### PR TITLE
Improve the computation of the reservation wage

### PR DESCRIPTION
**Pull request by TaichiSakaguchi**
Use `numpy.searchsorted` (which uses binary search) instead of the linear search in the original content.

### CODE REVIEW

File: lectures/mccall_model_with_separation.md [hunk 1]
```diff
@@ -438,11 +438,8 @@ def compute_reservation_wage(mcm):
     v, d = solve_model(mcm)
     h = u(mcm.c) + mcm.β * d
 
-    w_bar = np.inf
-    for i, wage in enumerate(mcm.w):
-        if v[i] > h:
-            w_bar = wage
-            break
+    i = np.searchsorted(v,h,side = 'right')
```
Comment (jstac) [inline lectures/mccall_model_with_separation.md]: Please add spaces after commas, no space around `=`.

**Comment by jstac** [comment]
Thanks @TaichiSakaguchi !  Please see the small request above.

**Comment by TaichiSakaguchi** [comment]
I fixed the code as you requested.

**Comment by jstac** [comment]
Thanks @TaichiSakaguchi

**Comment by jstac** [comment]
@mmcky , could you please look at this when you have time?  No rush.  The build error says `"level":"error","message":"token not found","stack":"Error: token not found\n...`

**Comment by mmcky** [comment]
thanks @jstac sorry I missed this one. 

The build is failing due to the `fork` origin. It's a frustrating limitation but it is due to the need to use GitHub credentials. 

I don't see an easy fix for these runner issues but I am looking into it. If the code looks good to go -- I think we can merge and the cache build will verify everything executes nicely.

**Comment by jstac** [comment]
Roger that @mmcky , merging.

**Pull request merged**